utils/clean_txt.py

```diff
@@ -2,59 +2,108 @@
 """A command line utility to clean text files generated from PDFs for Praxis-NLP."""
 import argparse
 from pathlib import Path
-from sys import stderr
 import re
 
-BRACKET_REGEX = re.compile(r"\s?\[\w*\]\s?|\s?\(\w*\)\s?")
-BIB_REGEX = re.compile(r"(Bibliography)|(Works Cited)|(Source Extracts)", re.IGNORECASE)
+# match expressions in square or round brackets containing alphanumeric
+# characters or spaces
+BRACKET_REGEX = re.compile(r'\s?\[(\w|\s|,)*\]|\s?\((\w|\s|,)*\)')
+BIB_REGEX = re.compile(
+    r'(Bibliography)|(Works Cited)|(Source Extracts)|(Appendix(:|\n))', re.IGNORECASE
```

Comment:
We may want to add References to this regex as well, seemed like a common pattern found in the 2T2 design briefs.